.idea/workspace.xml

```diff
@@ -11,6 +11,11 @@
     <option name="LAST_RESOLUTION" value="IGNORE" />
   </component>
   <component name="Git.Settings">
+    <option name="RECENT_BRANCH_BY_REPOSITORY">
```

Comment:
Eliminate the addition of unnecessary environment files using the .gitignore mechanism

Reply:
I mean .idea forlder should be ignored

Reply:
fix in new commit